docs/assets/style/theme.css

```diff
@@ -1,20 +0,0 @@
-.md-header,
```

Comment:
it's confusing to have a CSS in the dsp-tools repo, because docs.dasch.swiss has its own CSS. The one here will never be used in real life, only for local viewing. And for local viewing it's better to use the CSS from docs.dasch.swiss